internal/jujuapi/access_control.go

```diff
@@ -44,7 +33,7 @@ func (r *controllerRoot) AddGroup(ctx context.Context, req apiparams.AddGroupReq
 		return resp, errors.E(op, errors.CodeBadRequest, "invalid group name")
 	}
 
-	groupEntry, err := r.jimm.AddGroup(ctx, r.user, req.Name)
+	groupEntry, err := r.jimm.GetGroupManager().AddGroup(ctx, r.user, req.Name)
```

Comment:
what if 
```
r.jimm.GetGroupManager()
```
returns nil?

Reply:
I consider that the same as if the `r.jimm.db` is nil or `r.jimm.openFGAClient` is nil. Just let it panic because at that point we messed up. For now, that's more possible to happen because we don't have a jimm constructor but once your PR lands then we have a stronger guarantee this can't be nil.